SupportBank/Base.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using CsvHelper;
+
+namespace SupportBank
+{
+    class Base
+    {
+        public Dictionary<string,Account> Accounts { get; private set; } = new Dictionary<string,Account>();
```

Comment:
Minor but there should be whitespace between `string` and `Account`. And there should be an empty line between this property and the method below 🙂